.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
           go-version: 1.20.6
 
         - karmada-version: release-1.8
-          go-version: 1.20.11
+          go-version: 1.21.8
```

Comment:
Hi~ @Ray-D-Song thanks for your work. Since the latest version of karmada is `release-1.9`, we may need update the karmada version first, like:
```yaml
        karmada-version: [ release-1.7, release-1.8, release-1.9 ]
        include:
        - karmada-version: release-1.7
          go-version: 1.20.6

        - karmada-version: release-1.8
          go-version: 1.20.11

        - karmada-version: release-1.9
          go-version: 1.21.8
```

Reply:
> Hi~ @Ray-D-Song thanks for your work. Since the latest version of karmada is `release-1.9`, we may need update the karmada version first, like:
> 
> ```yaml
>         karmada-version: [ release-1.7, release-1.8, release-1.9 ]
>         include:
>         - karmada-version: release-1.7
>           go-version: 1.20.6
> 
>         - karmada-version: release-1.8
>           go-version: 1.20.11
> 
>         - karmada-version: release-1.9
>           go-version: 1.21.8
> ```

Thank you for reminding me, I will do it right away.

Reply:
Thanks @zhzhuang-zju for the reminder, I realized that too, but I didn't mention it as it is out of this PR's scope. But I'm OK to do that in this PR. Thanks @Ray-D-Song.